components/NavBar.tsx

```diff
@@ -84,20 +84,23 @@ function NavBar() {
                     GET STARTED
                   </Link>
                 </MenuItem>
-                {/* <MenuItem>
+                <MenuItem>
                   <Link
                     as={NextLink}
-                    href="/lessons"
+                    href="https://airtable.com/appDMMIARfSeiovpk/shrZExypPetXEx6Ox"
+                    target="_blank"
                     passHref
                     variant={
-                      router.pathname.startsWith('/lessons')
+                      router.pathname.startsWith(
+                        'https://airtable.com/appDMMIARfSeiovpk/shrZExypPetXEx6Ox',
+                      )
```

Comment:
this variant ternary works for menu buttons, as this is an external link the ternary never will be true.

can you just set the variant to 'top-navigation' ? thanks!!